fsspec/spec.py

```diff
@@ -196,6 +197,17 @@ def _strip_protocol(cls, path):
         # use of root_marker to make minimum required path, e.g., "/"
         return path or cls.root_marker
 
+    def _unstrip_protocol(self, name):
```

Comment:
Can we make this an exported method? (without _) since it's very useful in conjunction with url_to_fs